### PR TITLE
Coerce location metadata to unicode in fixture

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://github.com/dimagi/commcare-hq/pull/14799#issuecomment-278372147
@czue @emord

### CODE REVIEW

File: corehq/apps/locations/tests/test_location_fixtures.py [hunk 1]
```diff
@@ -91,6 +91,26 @@ def test_no_user_locations_returns_empty(self):
...
+        location_type = self.location_types['state']
+        location = SQLLocation(
+            id="854208",
+            domain="test-domain",
+            name="Braavos",
+            location_type=location_type,
+            metadata={
+                'best_swordsman': "Sylvio Forel",
+                'in_westeros': "false",
+                'appeared_in_num_episodes': 2,
```
Comment (esoergel) [inline corehq/apps/locations/tests/test_location_fixtures.py]: Actually he's in 3 episodes...

**Review by esoergel — CHANGES_REQUESTED** [review]

File: corehq/apps/locations/tests/test_location_fixtures.py (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/locations/tests/test_location_fixtures.py]: https://github.com/dimagi/commcare-hq/pull/14822/commits/81665f66885643e9e3c75396ecc2694c3ee271bf

**Review by esoergel — APPROVED** [review]

File: corehq/apps/locations/tests/test_location_fixtures.py (hunk 1 above)
Comment (czue) [inline corehq/apps/locations/tests/test_location_fixtures.py]: thank goodness for code review!

**Comment by kaapstorm** [comment]
This is probably going to resolve [FB 247293](http://manage.dimagi.com/default.asp?247293) too.

**Pull request merged**